bin/express

```diff
@@ -65,7 +66,7 @@ var www = fs.readFileSync(__dirname + '/../templates/js/www', 'utf-8');
     if (empty || program.force) {
       createApplicationAt(path);
     } else {
-      program.confirm('destination is not empty, continue? ', function(ok){
+      prompt.confirm('destination is not empty, continue? ')(function(err,ok){
```

Comment:
@rlidwka @dougwilson I agree that err should be handled. I am indirectly handling err in my code, such that it would fall thru the else part and abort.

Feel free to implement, you think is the right way. 
Cheers.